client/webserver/site/src/js/locales.ts

```diff
@@ -262,7 +266,12 @@ export const enUS: Locale = {
   [ID_ORDER]: 'Order',
   [ID_LOCKED_ORDER_BAL_MSG]: 'Funds locked in unmatched orders',
   [ID_CREATING_WALLETS]: 'Creating wallets',
-  [ID_ADDING_SERVERS]: 'Connecting to servers'
+  [ID_ADDING_SERVERS]: 'Connecting to servers',
+  [ID_INVALID_SEED]: 'Invalid seed',
+  [ID_EXPORT_SEED_DOWNLOAD]: 'Enter your app password to download your seed. Make sure the downloaded file is saved in a secure folder.',
+  [ID_EXPORT_SEED_VIEW]: 'Enter your app password to view your seed. Make sure nobody else can see your screen.',
+  [ID_DOWNLOAD]: 'Download',
```

Comment:
Reading downloading/uploading seed as file - it isn't something I personally would ever use, and it kinda promotes bad practice of storing uncyphered/unencrypted seed on your PC, or am I missing something ?

Reply:
> it kinda promotes bad practice of storing uncyphered/unencrypted seed on your PC

I agree. I was playing with the thought that downloading would be better than copying them(we currently don't restrict this). Manually writing out the seed hex can be a pita considering that they are not valid words even as they appear in chunks.

Reply:
> I was playing with the thought that downloading would be better than copying them(we currently don't restrict this).

I think @martonp implemented some kind of warning for using clipboard when dealing with private key for Ethereum, perhaps we can add similar warning for seed. But, "proper solution", as per your comment

> Manually writing out the seed hex can be a pita considering that they are not valid words even as they appear in chunks.

would be to move to 12/24 word seed phrase, I have no idea why it hasn't been done yet.

Reply:
Thanks for your input. I think these suggestions can go into an issue page for better attention.

Reply:
Yeah, this one's a pita because we have 64-bit seeds. Solution is needed. Keep clipboard available for now. OK to add a warning.